docs/source/how-to/developing-substra/harbor.rst

```diff
@@ -0,0 +1,121 @@
+**************************
+Harbor in local deployment
+**************************
+
+
+
+This page describes how to use Harbor in our development setup, using skaffold. It is divided in two categories: the first one describes how to use a harbor registry inside the `k3d` cluster, meanwhile the second describes how to use a harbor registry outside of the cluster. The thrid section describes the set-up that will be needed.
+
+
+
+Use in-cluster Harbor
+=====================
+
+First-time set-up (specific to in-cluster)
+------------------------------------------
+
+As ``sudo``, add ``127.0.0.1       harbor.harbor.svc.cluster.local`` to ``/etc/hosts``
+
+      .. code:: bash
+
+         echo "127.0.0.1       registry.org-2.com" | sudo tee -a /etc/hosts
```

Comment:
👀 
I think that the command does not do what the text says.

Reply:
Thanks, the second part is the correct one